shared/api-spec/geocode-response.json

```diff
@@ -17,7 +14,7 @@
         }
       },
       "additionalProperties": false,
-      "required": ["status", "lat", "lng"]
```

Comment:
You still need the status to know if geocoding failed though

Reply:
Good catch - updated